package.json

```diff
@@ -40,6 +41,7 @@
   },
   "dependencies": {
     "@elastic/elasticsearch": "^7.1.0",
+    "dotenv": "^8.0.0",
```

Comment:
make sure the types package (`^6.1.1`) is compatible with the implementation package (`^8.0.0`)